packages/sanity/src/core/tasks/plugin/TasksFooterOpenTasks.tsx

```diff
@@ -25,30 +25,28 @@ const ButtonContainer = styled.div`
 export function TasksFooterOpenTasks() {
   const {data, activeDocument} = useTasks()
   const {handleOpenTasks, setActiveTab} = useTasksNavigation()
-  const {enabled} = useTasksEnabled()
-
   const mediaIndex = useMediaIndex()
-
   const pendingTasks = useMemo(
     () =>
-      data.filter((item) => {
-        return (
-          item.target?.document._ref === activeDocument?.documentId &&
-          item.status === 'open' &&
-          item.createdByUser
-        )
-      }),
+      activeDocument?.documentId
+        ? data.filter((item) => {
+            return (
+              item.target?.document._ref === activeDocument?.documentId &&
```

Comment:
COULD: get rid of the optional chain here since `activeDocument` will always be defined in this block

Reply:
Sure! Thanks for noticing.